chall07/pcuadrad.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"strconv"
+	"log"
+	"os"
+	"time"
+)
+
+func RGBToHex(r int, g int, b int) (string) {
+	return fmt.Sprintf("%x%x%x",r ,g ,b)
+}
+
+func printRequestTime(url string, answer string, start time.Time) {
+	now := time.Now()
+	elapsed := now.Sub(start)
+	if answer == "" {
+		fmt.Println(elapsed,"- GET", url)
+	} else {
+		fmt.Println(elapsed,"- GET", url, "\n\tanswer:", answer)
+	}
```

Comment:
I really like the way you tried to respect the examples' output, I advise you looking at the builtin [Log Package](https://golang.org/pkg/log/) to make better logs. _(I noticed later that you used it, it is better to print logs with the same library,  personally, for big project I use `fmt.Print` functions to debug, but I never let them in my code.)_  
Or even, my favorite log package: [Logrus](https://github.com/sirupsen/logrus), it has many useful features, and a really nice formatting.

Reply:
Okay, thanks for the recommendation!